vnext/ReactUWP/Views/TextViewManager.cpp

```diff
@@ -179,5 +184,23 @@ YGMeasureFunc TextViewManager::GetYogaCustomMeasureFunc() const {
   return DefaultYogaSelfMeasureFunc;
 }
 
+void TextViewManager::OnDescendantTextPropertyChanged(ShadowNodeBase *node) {
+  if (auto element = node->GetView().try_as<winrt::TextBlock>()) {
+    // If name is set, it's controlled by accessibilityLabel, and it's already
+    // handled in FrameworkElementViewManager. Here it only handles when name is
+    // not set.
+    if (winrt::AutomationProperties::GetLiveSetting(element) !=
+            winrt::AutomationLiveSetting::Off &&
+        winrt::AutomationProperties::GetName(element).empty() &&
+        winrt::AutomationProperties::GetAccessibilityView(element) !=
+            winrt::Peers::AccessibilityView::Raw) {
+      if (auto peer =
+              winrt::FrameworkElementAutomationPeer::FromElement(element)) {
+        peer.RaiseAutomationEvent(winrt::AutomationEvents::LiveRegionChanged);
```

Comment:
Did you confirm when the event is raised here, that narrator actually announces the text?

Reply:
yes. I did manual testing